app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.java

```diff
@@ -127,6 +128,7 @@
 				new AddMotorcycleParkingCapacity(o),
 				new AddPathSurface(o),
 				new AddTracktype(o),
+				new AddLeafType(o), // used by OSM Carto
```

Comment:
maybe rename to `AddWoodLeafType` to leave room for `AddTreeLeafType`

Reply:
Renamed (though to `AddForestLeafType`)